utils/wasm-proc/src/main.rs

```diff
@@ -158,6 +160,16 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             continue;
         }
 
+        let res = gear_wasm_builder::optimize::optimize_wasm(PathBuf::from(file), "s", true)
```

Comment:
I think it should be forced otherwise we won't have a culture of program size optimization.

> Or how about to provide such optimization functionality just inside the gear wasm builder?

I'll look into it but I think it's the simplest solution and we don't need to reinvent the wheel.

Reply:
Ok, but I want you to open issue for consider about inclusion of opt functionality directly in builder